capture.sh

```diff
@@ -11,9 +20,28 @@ else
   exit 1
 fi
 
+PERIODI=4 #default to 4 seconds between captures. Can be changed with --period command line option
+if [ ! -z "${period}" ]
+    then
+      PERIODI="$period"
+fi
+
+if [ ! -z "${app}" ] && test "$(uname)" != 'Darwin'
+    then
+      echo 'The --app option is currently only available on macOS.'
+      exit 0
+fi
+
 # screen captures
 while [ 1 ]; do
-  $SCREENCAPTURE_CMD ./images/`date +%s`.jpg
-  echo "captured `date +%s`"
-  sleep 4
+  if [ ! -z "${app}" ]
+      then
+        APPINFOCUS=`osascript -e 'tell application "System Events"' -e 'set frontApp to name of first application process whose frontmost is true' -e 'end tell'`
+  fi
+  if [ "${APPINFOCUS}" == "${app}" ] || [ -z "${app}" ] #if --app command line option is provided, only capture when the specified app is in focus
```

Comment:
Nit: space after comment

```suggestion
  if [ "${APPINFOCUS}" == "${app}" ] || [ -z "${app}" ] # if --app command line option is provided, only capture when the specified app is in focus
```